### PR TITLE
fix(stepper): fixed stepper as per the design

### PR DESCRIPTION
**Pull request by tricaalexandru**
# <JIRA issue / Title>

## TL;DR

-   List
-   The
-   Core
-   Changes

## Details

Checklist / Reminders:

-   [x] Documentation / Story is created / adapted
-   [x] Unit tests
-   [x] CSS changes are BEM-compliant

### CODE REVIEW

**Review by philjonasfiltered — APPROVED** [review]

File: src/components/Molecules/Steps/Steps.interface.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { ReactElement } from 'react'
 import { Elevation, ThemeColors } from '@/types/theme'
 
-export type StepsVariant = 'prominent' | 'simple'
+export type StepsVariant = 'prominent' | 'simple' | 'equidistant'
```
Comment (matthill8286) [inline src/components/Molecules/Steps/Steps.interface.ts:4]:
```suggestion
export type StepsVariant = 'prominent' | 'simple' | 'uniform'
```

File: src/components/Molecules/Steps/Steps.story.tsx [hunk 2]
```diff
@@ -40,7 +41,7 @@ stories.add(
       color: select('color', colors, 'primary'),
       activeStep: select('activeStep', stepsIndex, 2),
       completedStep: select('completedStep', [-1, ...stepsIndex], 1),
-      variant: radios('variant', variants, 'prominent'),
+      variant: radios('variant', variants, 'equidistant'),
```
Comment (matthill8286) [inline src/components/Molecules/Steps/Steps.story.tsx:44]:
```suggestion
      variant: radios('variant', variants, 'uniform'),
```

File: src/components/Molecules/Steps/Steps.story.tsx [hunk 3]
```diff
@@ -184,6 +185,34 @@ stories.add(
   }
 )
 
+stories.add(
+  'Equidistant',
```
Comment (matthill8286) [inline src/components/Molecules/Steps/Steps.story.tsx:189]:
```suggestion
  'Uniform',
```

File: src/components/Atoms/Button/Button.story.tsx [hunk 4]
```diff
@@ -101,6 +101,19 @@ storiesOf('Design System/Atoms/Button', module)
...
+      return (
+        <Button curved size={'lg'} {...knobs()}>
+          Button text
+        </Button>
+      )
+    },
+    {
+      info: Readme,
+    }
+  )
```
Comment (matthill8286) [inline src/components/Atoms/Button/Button.story.tsx:116]: This is covered in Onboarding button as it's the only place it's used. Other buttons are generic 👍

**Review by matthill8286 — APPROVED** [review]

**Comment by tricaalexandru** [comment]
@matthill8286 sweet! Updated as per review

**Pull request merged**